env0/resource_project_policy.go

```diff
@@ -93,6 +93,18 @@ func resourcePolicy() *schema.Resource {
 				Description: "updated by",
 				Computed:    true,
 			},
+			"run_pull_request_plan_default": {
+				Type:        schema.TypeBool,
+				Description: "Run Terraform Plan on Pull Requests for new environments targeting their branch default value",
+				Optional:    true,
+				Default:     false,
```

Comment:
Since all other fields here don't have a default value setting, I think we should change it to either make all of them with default values, or none.

Reply:
https://github.com/env0/terraform-provider-env0/pull/164/commits/485d952db9ad33f6af7c279d97455bf0b0912b2c